### PR TITLE
docs(foundations/color): grammar review

### PR DESCRIPTION
**Pull request by mohamedMok**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

GitHub issue number or Jira issue URL: N/A

### CODE REVIEW

File: src/docs/Foundations/Colors/index.mdx [hunk 1]
```diff
@@ -67,21 +67,21 @@ Inform the user that an operation was successful.
 
 <Swatch id="success" />
 
-## Warning
+## Warnings
 
-Used to inform abouts risks or threats. It teel the user to be carefull about something.
+Inform the user about risks or threats. It teel the user to be careful about something.
```
Comment (nherwegh) [inline src/docs/Foundations/Colors/index.mdx:72]: It tells

File: src/docs/Foundations/Colors/index.mdx [hunk 2]
```diff
@@ -67,21 +67,21 @@ Inform the user that an operation was successful.
...
 
 <Swatch id="warning" />
 
 ## Danger
 
-Used to inform user of an error, or that some operation whent wrong.
+Inform the user of an error, or that some operation went wrong.
```
Comment (nherwegh) [inline src/docs/Foundations/Colors/index.mdx:78]: Informs the user of an error or that some operation went wrong.

File: src/docs/Foundations/Colors/index.mdx [hunk 3]
```diff
@@ -92,13 +92,13 @@ Use the [info]() swatch to highlight anything related to informational feedback
 
 ## Secondary Green
 
-Secondary Green is for success. In LEROY MERLIN case, it is exacltly the case as the primary-01. But other brands or BU should be able to change that.
+Secondary Green is for success. In LEROY MERLIN case, it is exactly the case as the primary-01. But other brands or BU should be able to change that.
```
Comment (nherwegh) [inline src/docs/Foundations/Colors/index.mdx:95]: In LEROY MERLIN case, this color is exactly the case as `primary-01` but other brands or BU should be able to change that.

File: src/docs/Foundations/Colors/index.mdx [hunk 4]
```diff
@@ -124,11 +124,11 @@ Secondary Purple is used for illustration purposes.
 
 In this system, every color has a `dark` and a `light` variant that can be used to adapt to different usages from text to illustrations.
 
-In order to have this flexibility we designed color swatches based on **9 values** for primary and secondary colors. It means that you can use 9 values of the same color to fit your design needs.
+To have this flexibility, we designed color swatches based on **9 values** for primary and secondary colors. It means that you can use 9 values of the same color to fit your design needs.
 
-> HSLA is a way to define colors based on three properties : **Hue**, **Saturation**, **Light** and **Alpha**. We decided to use this standard to have more consistency between colors. It is also easier to relate colors to each other when they are based on HSLA.
+> HSLA is a way to define colors based on three properties: **Hue**, **Saturation**, **Light**, and **Alpha**. We decided to use this standard to have more consistency between colors. It is also easier to relate colors to each other when they are based on HSLA.
```
Comment (nherwegh) [inline src/docs/Foundations/Colors/index.mdx:129]: I think we can delete this part
@caqueste, we're not making rockets fly right?

File: src/docs/Foundations/Colors/index.mdx [hunk 5]
```diff
@@ -124,11 +124,11 @@ Secondary Purple is used for illustration purposes.
...
 
-This system allows the team to have every tint it needs to design every situation. It answers typography problems, illustration needs and it also allows to contrast with deep background colors. Here is an example showing how it works based on our Primary-01 Secondary-Purple colors.
+This system allows the team to have every tint it needs to design every situation. It answers typography problems, illustration needs and it also allows us to contrast with deep background colors. Here is an example showing how it works based on our Primary-01 Secondary-Purple colors.
```
Comment (nherwegh) [inline src/docs/Foundations/Colors/index.mdx:131]: Delete this also? @caqueste

File: src/docs/Foundations/Colors/index.mdx [hunk 6]
```diff
@@ -1,48 +1,48 @@
...
 order: 3
-description: 'Mozaic offers an adaptive and a scalable color system. The color palette has been divided in three categories: primary colors, secondary colors and greys.'
+description: 'Mozaic offers an adaptive and scalable color system. The color palette has been divided into three categories: primary colors, secondary colors, and greys.'
 status:
   sketch: 'stable'
   scss: 'stable'
 ---
 
-> Mozaic offers an adaptive and a scalable color system. These colors have been defined using [HSLA](/Foundations/Colors/#anatomy) color swatches. The color palette has been divided in four categories : **Primary colors**, **Secondary colors** ,**Greys** and **Status colors**.
+> Mozaic offers an adaptive and scalable color system. These colors have been defined using [HSLA](/Foundations/Colors/#anatomy) color swatches. The color palette has been divided in four categories: **Primary colors**, **Secondary colors**,**Greys** and **Status colors**.
```
Comment (nherwegh) [inline src/docs/Foundations/Colors/index.mdx:10]: Mozaic offers an adaptive and scalable color system. The color palette has been divided in four categories: **Primary colors**, **Secondary colors**,**Greys** and **Status colors**.

@caqueste agree with that?

**Review by nherwegh — CHANGES_REQUESTED** [review]

File: src/docs/Foundations/Colors/index.mdx (hunk 4 above)
Comment (caqueste) [reply src/docs/Foundations/Colors/index.mdx:129]: @nherwegh @mohamedMok I think this part can save us from the covid..
Let's delete it haha

File: src/docs/Foundations/Colors/index.mdx (hunk 5 above)
Comment (caqueste) [reply src/docs/Foundations/Colors/index.mdx:131]: @nherwegh Yes, we can delete this part too, I think the visuals are enough explicit.

File: src/docs/Foundations/Colors/index.mdx (hunk 6 above)
Comment (caqueste) [reply src/docs/Foundations/Colors/index.mdx:10]: @nherwegh I love it!

File: src/docs/Foundations/Colors/index.mdx [hunk 7]
```diff
@@ -52,36 +52,36 @@ We used the same principles to design our greyscale but greys are based on **11
...
 
-Used to inform abouts risks or threats. It teel the user to be carefull about something.
+Inform the user about risks or threats. It teels the user to be careful about something.
 
 <Swatch id="warning" />
 
 ## Danger
 
-Used to inform user of an error, or that some operation whent wrong.
+Inform the user of an error or that some operation went wrong.
```
Comment (nherwegh) [inline src/docs/Foundations/Colors/index.mdx:78]: This color informs the user of an error or that some operation went wrong. 

(I think it's better for the global understanding 😉)

**Review by nherwegh — CHANGES_REQUESTED** [review]

**Review by nherwegh — APPROVED** [review]

File: src/docs/Foundations/Colors/index.mdx [hunk 8]
```diff
@@ -1,48 +1,48 @@
 ---
 title: 'Colors'
 order: 3
-description: 'Mozaic offers an adaptive and a scalable color system. The color palette has been divided in three categories: primary colors, secondary colors and greys.'
+description: 'Mozaic offers an adaptive and scalable color system. The color palette has been divided into three categories: primary colors, secondary colors, and greys.'
```
Comment (caqueste) [inline src/docs/Foundations/Colors/index.mdx:4]: ~~Mozaic offers an adaptive and scalable color system. The color palette has been divided into three categories: primary colors, secondary colors, and greys.~~

Mozaic offers an adaptive and scalable color system. The color palette has been divided into four categories: Primary colors, Secondary colors, Greys and Status colors.

File: src/docs/Foundations/Colors/index.mdx [hunk 9]
```diff
@@ -1,48 +1,48 @@
...
 status:
   sketch: 'stable'
   scss: 'stable'
 ---
 
-> Mozaic offers an adaptive and a scalable color system. These colors have been defined using [HSLA](/Foundations/Colors/#anatomy) color swatches. The color palette has been divided in four categories : **Primary colors**, **Secondary colors** ,**Greys** and **Status colors**.
+> Mozaic offers an adaptive and scalable color system. The color palette has been divided in four categories: **Primary colors**, **Secondary colors**,**Greys** and **Status colors**.
```
Comment (caqueste) [inline src/docs/Foundations/Colors/index.mdx:10]: ~~in~~ into

**Review by caqueste — CHANGES_REQUESTED** [review]

**Review by caqueste — APPROVED** [review]

**Pull request merged**